### ISSUE TITLE
.validate() false positive

### ISSUE THREAD

**Issue by jan-macek** [NaturalIntelligence/fast-xml-parser#336]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [x] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Input
```
<audio src="https://example.com/example.mp3"/>text
```

### Code
```js
const parser = require('fast-xml-parser');
console.log(parser.validate('<audio src="https://example.com/example.mp3"/>text'));
```

### Output
```
true
```

### expected data
```
false
```

**Would you like to work on this issue?**

- [ ] Yes
- [x] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.

**Comment by amitguptagwl** [comment]
fixed in v4
